src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.cs

```diff
@@ -139,10 +139,10 @@ internal static string GetNormalizedStrongNameHash(AssemblyName name)
                     bw.Write(publicKey);
                     bw.Write(name.Version.Major);
                     bw.Write(name.Name);
-                }
 
-                ms.Position = 0;
-                return GetStrongHashSuitableForObjectName(ms);
```

Comment:
Why this change?  Does BinaryWriter implicitly flush to the underlying stream on every write?